cli/src/pcluster/config/extended_builtin_class.py

```diff
@@ -51,3 +51,13 @@ def __init__(self, b: bool, implied: bool = True):
     def __bool__(self):
         """Bool operation depends on object attribute b."""
         return self.b
+
+
+class MarkedTag(object):
+    """Custom class to add more attributes to the tag."""
+
+    def __init__(self, key: str, value: str, implied: bool = True):
+        """Initialize extended tag."""
+        self.key = key
+        self.value = value
+        self.implied = implied
```

Comment:
I am not sure about this. If in the future we add more attributes and methods to `Tag` class, there will be inconsistencies between the `Tag` and `MarkedTag`.

Reply:
use Tag with extra attribute `implied` rather than MarkedTag